Quantum-Control-Applications/Superconducting/Single Fixed Transmon/configuration.py

```diff
@@ -89,17 +89,15 @@ def readout_macro(threshold=None, state=None, I=None, Q=None):
 x180_len = 40
 x180_sigma = x180_len / 5
 x180_amp = 0.35
-x180_drag_wf, x180_drag_der_wf = np.array(
-    drag_gaussian_pulse_waveforms(x180_amp, x180_len, x180_sigma, alpha=0, delta=1)
-)
+x180_wf, x180_der_wf = np.array(drag_gaussian_pulse_waveforms(x180_amp, x180_len, x180_sigma, alpha=0, delta=1))
 # No DRAG when alpha=0, it's just a gaussian.
 
-x90_len = x180_len
+x90_len = x270_len = y90_len = y180_len = y270_len = x180_len
 x90_sigma = x90_len / 5
 x90_amp = x180_amp / 2
-x90_drag_wf, x90_drag_der_wf = np.array(drag_gaussian_pulse_waveforms(x90_amp, x90_len, x90_sigma, alpha=0, delta=1))
+x90_wf, x90_der_wf = np.array(drag_gaussian_pulse_waveforms(x90_amp, x90_len, x90_sigma, alpha=0, delta=1))
 # No DRAG when alpha=0, it's just a gaussian.
-
+x270_wf, x270_der_wf = np.array(drag_gaussian_pulse_waveforms(-x90_amp, x90_len, x90_sigma, alpha=0, delta=1))
```

Comment:
If we have `x270_len`, let's use it...
So either:
1. Remove `x270_len` above
2. Keep `x270_len`, but move it down (line 100), also add `x270_amp=-x90_amp` and `x90_sigma`. Redefine the drag waveform to use these variables.

Also, we're not using all of the y lens, so let's remove them...